app/views/email_templates/new.html.slim

```diff
@@ -1,32 +1,29 @@
 h1= t_title(:new)
 
-p Sie können die folgenden Platzhalter benützen:
-ul
-  - EmailTemplate.template_variables.each do |var|
-    li= "%{#{var}}"
-p Zum Beispiel:
-p Liebe/r %{Anrede} %{Name}
-p Wir bitten um Ihre %{FeedbackLink} für %{EinsatzTitel}.
-p Danke im Voraus!
-
 = render 'form'
 
-.load-templates
-  - EmailTemplate.active.each do |template|
-    = hidden_field_tag "subjects[#{template.kind}]", template.subject
-    = hidden_field_tag "bodies[#{template.kind}]", template.body
-
-= form_navigation_btn :back
-
 javascript:
-  $(document).on('turbolinks:render, turbolinks:load', function() {
-    $('#email_template_kind').change(function(){
-      $("#email_template_subject").text("");
-      $("#email_template_body").text("");
-      var kind = $("#email_template_kind").val();
-      var subject = $("input[TYPE=HIDDEN]#subjects_" +kind).val();
-      var body = $("input[TYPE=HIDDEN]#bodies_" +kind).val();
+  $(document).on('turbolinks:render, turbolinks:load', function () {
```

Comment:
So we will not provide initial Templates for each kind? I would rather consider to provide initial Templates for each kind, so that cannot happen, plus it would be just way more userfriendly (user sees how it could/should be done)